packages/cosmic-swingset/test/make.test.ts

```diff
@@ -39,9 +32,10 @@ const makeTestContext = async t => {
     pspawnMake,
     pspawnAgd,
     log: t.log,
-    stdio: process.env.DEBUG ? fdList : undefined,
+    stdio: process.env.DEBUG ? fdList : undefined, // Use explicit stdio for debugging
```

Comment:
YMMV, but I don't find this comment to be helpful.
```suggestion
    stdio: process.env.DEBUG ? fdList : undefined,
```
or maybe
```suggestion
    stdio: process.env.DEBUG ? fdList : undefined, // propagate stdout and stderr for DEBUG
```